### PR TITLE
Sammysteiner/issue 96 frontend language

### LINKED ISSUE #96: ADR: Frontend Language

### PR DESCRIPTION
**Pull request by SammySteiner**
## Summary
Fixes #96

### CODE REVIEW

File: documentation/decisions/adr/0007-front-end-language.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+- JavaScript
+- Typescript
+- Python
+- Java
+- Go
+
+## Decision Outcome <!-- REQUIRED -->
+
+Chosen option: Typescript, because it is the ideal language for front-end applications because some form of JavaScript is required for client side functionality anyway. This allows us to reduce the amount of context switching between langauges. Additionally, with node and npm TypeScript and JavaScript are fully featured platforms for building and running applications. We chose TypeScript over JavaScript because of the value that strong typing provides with promomting code quality and developer support.
```
Comment (lucasmbrown-usds) [inline documentation/decisions/adr/0007-front-end-language.md:31]: I'm a big TypeScript fan! 

(also, minor nitpick: I think it's stylized with a capital S, just like JavaScript.)

Comment (lucasmbrown-usds) [reply documentation/decisions/adr/0007-front-end-language.md:31]: BTW, I've also been a big fan of enforcing immutability in JS to reduce bugs, e.g. by using https://immutable-js.com/ and a linter to yell at you if you try to mutate state. But that's a conversation for a later milestone, not this one. Just planting the seed for consideration.

**Review by lucasmbrown-usds — APPROVED** [review]
LGTM! Well-documented.

File: documentation/decisions/adr/0007-front-end-language.md (hunk 1 above)
Comment (SammySteiner) [reply documentation/decisions/adr/0007-front-end-language.md]: Thanks! 
Yes, thanks for catching that. I missed a couple `javascript's too, but it should be good now.

**Pull request merged**